recipe/0001-Link-to-abseil_dll.patch

```diff
@@ -1,17 +1,35 @@
-From b2828440f9930c76e42235ddccfa0954d58a84ac Mon Sep 17 00:00:00 2001
+From bcfcda2ddb3738acec2a96938060da9b16408d21 Mon Sep 17 00:00:00 2001
 From: "Uwe L. Korn" <uwe.korn@quantco.com>
 Date: Tue, 4 Jul 2023 11:29:21 +0200
 Subject: [PATCH] Link to abseil_dll
 
 ---
- CMakeLists.txt | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
+ CMakeLists.txt | 5 ++++-
+ 1 file changed, 4 insertions(+), 1 deletion(-)
 
 diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 4dd56b6..3a002a5 100644
+index 0372c54b..1224759c 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -430,7 +430,7 @@ target_include_directories(onnx PUBLIC
+@@ -504,14 +504,17 @@ target_compile_definitions(onnx_proto PRIVATE ${ONNX_API_DEFINE})
+ if(ONNX_USE_LITE_PROTO)
+   if(TARGET protobuf::libprotobuf-lite)
+     target_link_libraries(onnx_proto PUBLIC protobuf::libprotobuf-lite PRIVATE ${protobuf_ABSL_USED_TARGETS})
++    target_link_libraries(onnx_proto PUBLIC $ENV{PREFIX}/Library/lib/abseil_dll.lib)
+   else()
+     target_link_libraries(onnx_proto PUBLIC ${PROTOBUF_LITE_LIBRARIES})
+   endif()
+ else()
+   if(TARGET protobuf::libprotobuf)
+     target_link_libraries(onnx_proto PUBLIC protobuf::libprotobuf PRIVATE ${protobuf_ABSL_USED_TARGETS})
++    target_link_libraries(onnx_proto PUBLIC $ENV{PREFIX}/Library/lib/abseil_dll.lib)
+   else()
+     target_link_libraries(onnx_proto PUBLIC ${PROTOBUF_LIBRARIES})
++    target_link_libraries(onnx_proto PUBLIC $ENV{PREFIX}/Library/lib/abseil_dll.lib)
+   endif()
+ endif()
+ add_onnx_global_defines(onnx_proto)
+@@ -528,7 +531,7 @@ target_include_directories(onnx PUBLIC
```

Comment:
If you find protobuf to `find_package(Protobuf CONFIG)`, this should be handled automatically by CMake.

Reply:
Anyhow, if this works fine it is ok to use that.